src/error.rs

```diff
@@ -118,6 +118,9 @@ pub enum CmdError {
     /// Could not decode a base64 image
     ImageDecodeError(base64::DecodeError),
 
+    /// Could not decode a base64 PDF.
+    PdfDecodeError(base64::DecodeError),
```

Comment:
Ooof, this is a breaking change. I thought we had already marked this `enum` as `#[non_exhaustive]`, but apparently not :disappointed: Would you mind adding that annotation as well?